Lib/ufo2ft/filters/optimizeAnchors.py

```diff
@@ -0,0 +1,27 @@
+from ufo2ft.filters.transformations import TransformationsFilter
+from fontTools.misc.transform import Identity, Transform
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class OptimizeAnchorsFilter(TransformationsFilter):
+    def set_context(self, font, glyphSet):
+        # Skip over transformations filter to base filter
+        return super(TransformationsFilter, self).set_context(font, glyphSet)
```

Comment:
better to explicitly call the `BaseFilter.set_context(self, font, glyphSet)` since that is what you are after.